slate-video-library/src/App.js

```diff
@@ -1,13 +1,19 @@
 import React from "react";
 import { Route, Routes } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
 import "./App.css";
+import Authentication from "./Components/Authentication/Authentication";
+import Pagenotfound from "./Components/PageNotFound/Pagenotfound";
+import Accountpage from "./Pages/Accountpage/Accountpage";
 import Explorepage from "./Pages/Explorepage/Explorepage";
 import Historypage from "./Pages/Historypage/Historypage";
 import Homepage from "./Pages/Homepage/Homepage";
 import Likespage from "./Pages/Likespage/Likespage";
+import Loginpage from "./Pages/Loginpage/Loginpage";
 import Playlistspage from "./Pages/Playlistspage/Playlistspage";
 import PlaylistViewpage from "./Pages/Playlistspage/PlaylistViewpage";
 import SingleVideopage from "./Pages/SingleVideoPage/SingleVideopage";
+import Signuppage from "./Pages/Singuppage/Signuppage";

```

Comment:
Suggestion: export all pages and components from one file and then import it looks good.